installation fails

squid@kali:~/reaver-wps-fork-t6x/src# sudo make install
rm -f /usr/local/bin/reaver /usr/local/bin/wash /usr/local/bin/walsh
if [ ! -d /usr/local/etc/reaver ]; then mkdir -p /usr/local/etc/reaver; fi
if [ ! -e /usr/local/etc/reaver/reaver.db ]; then cp reaver.db /usr/local/etc/reaver/reaver.db && chmod -R a+rw /usr/local/etc/reaver; fi
if [ -e wash ]; then cp wash /usr/local/bin/wash; fi
cp: cannot create regular file '/usr/local/bin/wash': No such file or directory
Makefile:96: recipe for target 'install' failed
make: **\* [install] Error 1

kali distro
Linux version 4.3.0-kali1-amd64 (debian-kernel@lists.debian.org) (gcc version 5.3.1 20160121 (Debian 5.3.1-7) ) #1 SMP Debian 4.3.3-7kali2 (2016-01-27)

--- a/src/Makefile.in
+++ b/src/Makefile.in
@@ -99,10 +99,10 @@ cracker.o: globule.o init.o pins.o iface.o exchange.o session.o 80211.o
 globule.o:
 	$(CC) $(CFLAGS) globule.c -c 
 
-install: cleanbin
-	if [ ! -d $(CONFDIR) ]; then mkdir -p $(CONFDIR); fi
-	if [ -e wash ]; then cp wash @bindir@/wash; fi
-	if [ -e reaver ]; then cp reaver @bindir@/reaver; fi
+install: all
+	install -d $(DESTDIR)$(CONFDIR)
+	install -Dm 755 wash $(DESTDIR)@bindir@/wash
+	install -Dm 755 reaver $(DESTDIR)@bindir@/reaver
 
 clean:
 	rm -f *~ *.o reaver wash
@@ -114,10 +114,6 @@ clean:
 	(cd lwe && $(MAKE) realclean)
 	rm -f Makefile config.h
 
-cleanbin: 
-	rm -f @bindir@/reaver @bindir@/wash @bindir@/walsh
+distclean: clean
 
-distclean: cleanbin clean
-	rm -rf $(CONFDIR)
-
-.PHONY: libiw clean cleanbin distclean install
+.PHONY: libiw clean distclean install

